code/game/objects/effects/spawners/bombspawner.dm

```diff
@@ -1,110 +1,3 @@
-/* The old single tank bombs that dont really work anymore

```

Comment:
The last time this entire code block was changed was 7y ago, and its been commented out for 10y